doc/sample.rst

```diff
@@ -115,21 +116,20 @@ After programming the sample and all prerequisites to the development kit, test
 Troubleshooting
 ===============
 
-Q: I don't see any output log such as "... Next update check in...". I only see "SoftBank FOTA client started".
-
-A: By default, the library does not use LOG. Check that you have linked the alternative verbose library ``CONFIG_SB_FOTA_LOG=y``
+IF you do not see any output log such as "... Next update check in..." and only see "SoftBank FOTA client started", check that you have linked the alternative verbose library ``CONFIG_SB_FOTA_LOG = y``.
+By default, the library does not use LOG.
 
 Dependencies
 ************
 
 This sample uses the following |NCS| libraries:
 
-* nRF Cloud
-* (AT) Shell
+* `nRF Cloud`_
```

Comment:
This section is wrong and should have been removed earlier. nRF Cloud is not a NCS library and the sample doesn't use the AT Shell. (But it probably should do that instead of implementing its own AT shell command)

Reply:
Removed.